cpp/merkletree/tree_hasher.cc

```diff
@@ -1,31 +1,31 @@
 #include "merkletree/tree_hasher.h"
 
+#include <glog/logging.h>
+
 #include "merkletree/serial_hasher.h"
 
+using std::lock_guard;
+using std::mutex;
 using std::string;
 
-const string TreeHasher::kLeafPrefix(1, '\x00');
-const string TreeHasher::kNodePrefix(1, '\x01');
+namespace {
 
-TreeHasher::TreeHasher(SerialHasher* hasher) : hasher_(hasher) {
-}
+const string kLeafPrefix(1, '\x00');
```

Comment:
Could you make these `const char` to avoid the styleguide forbidden global var of a complex class?